rsb/__init__.py

```diff
@@ -2167,4 +2151,38 @@ def create_remote_server(scope, config=None, parent=None, **kwargs):
                               parent=parent, **kwargs)
 
 
-_register_default_transports()
+_plugins_loaded = False
+_plugins_lock = threading.RLock()
+
+
+def _load_plugins():
+    global _plugins_loaded
+
+    with _plugins_lock:
+        if _plugins_loaded:
+            return
+        _plugins_loaded = True
+
+        default_plugins = ['rsb.transport.local', 'rsb.transport.socket']
+        configured_plugins = [p.strip()
+                              for p in _default_configuration_options.get(
+                                  'plugins.python.load', '').split(':')]
+        configured_plugins = [p for p in configured_plugins if len(p) > 0]
```

Comment:
>It could, but I find this already hard enough to read.

In that case, why not a temporary for `_default_configuration_options.get('plugins.python.load', '').split(':')` and the list comprehension can do all the transforming and filtering?

Reply:
Better?